File: 2_Kernel/JavaKernel/src/main/java/com/seagosoft/tws/data/AccountUpdateMultiData.java
package com.seagosoft.tws.data;

public class AccountUpdateMultiData {

    public int reqId;
    public String account;
    public String modelCode;
    public String key;
    public String value;
    public String currency;

    public AccountUpdateMultiData(int reqId, String account, String modelCode,
                                  String key, String value, String currency) {
        this.reqId = reqId;
        this.account = account;
        this.modelCode = modelCode;
        this.key = key;
        this.value = value;
        this.currency = currency;
    }
}

File: 3_OrderSystem/KotlinEWrapperBackend/src/main/kotlin/com/seagosoft/tws/wrapper/structs/AccountUpdateMultiData.kt
package com.seagosoft.tws.wrapper.structs

class AccountUpdateMultiData(var reqId: Int, var account: String, var modelCode: String,
                             var key: String, var value: String, var currency: String) {
    override fun toString(): String {
        return "AccountUpdateMultiData(reqId=$reqId, account='$account', modelCode='$modelCode', key='$key', value='$value', currency='$currency')"
    }
}